### ISSUE TITLE
arrow function trigger "call-signature"

### ISSUE THREAD

**Issue by Fank** [palantir/tslint#1327]
### Bug Report

- __TSLint version__: `3.11.0`
- __TypeScript version__: `1.7.10`
- __Running TSLint via__: CLI / grunt-tslint / Atom <- all tested

#### TypeScript code being linted

```ts
class AComponent {
	private _test(): void {
		this._foo.test({
			callback: () => {
				// asd
			},
		});
	}
}
```

with `tslint.json` configuration:

```json
...
    "typedef": [
      true,
      "parameter",
      "call-signature"
    ],
...
```

#### Actual behavior

Linter say `callback: () => {` this line needs to have a `call-signature`

#### Expected behavior

Linter should ignore this, because `arrow-call-signature` is not set

**Comment by battmanz** [comment]
I was just about to enter this same issue. I would love to have this.

**Comment by JKillian** [comment]
@YuichiNukiyama @tomduncalf think this is related to #1176 or #1284? Probably #1284 caused it?

**Comment by tomduncalf** [comment]
Yeah I would guess #1284?

**Comment by YuichiNukiyama** [comment]
Hi, @Fank. I add tests to verify this issue. But, nothing has happened (see #1332).
I want to more details.

**Comment by JKillian** [comment]
Thanks for looking into this @YuichiNukiyama! It's strange that the tests aren't showing the same error. @Fank, what does your configuration look like?

**Comment by YuichiNukiyama** [comment]
I noticed that there is a simple oversight, #1176 and #1284 haven't inlcude in **v3.11** yet.

**Comment by Fank** [comment]
Yea this might be the problem :+1:

**Comment by Fank** [comment]
@JKillian is it possible to release 3.12 soon?

**Comment by adidahiya** [comment]
@Fank sure, we'll try to release v3.12 today.

**Comment by JKillian** [comment]
Published! Try out the new version and let us know if any problems persist

**Comment by Fank** [comment]
Ill do thanks
